trade-finance-manager-ui/server/controllers/helpers/isFacilityEndDateEnabledForFacility.js

```diff
@@ -0,0 +1,13 @@
+import { isTfmFacilityEndDateFeatureFlagEnabled } from '@ukef/dtfs2-common';
+
+/**
+ * Facilities are part of a GEF deal if and only if they have a cash or contingent facility type.
+ * In mapGefFacility.js, the received facility gets mapped from 'CASH' or 'CONTINGENT' to 'Cash facility' and 'Contingent facility'
+ */
+const isFacilityPartOfGefDeal = (type) => {
+  return type === 'Cash facility' || type === 'Contingent facility';
```

Comment:
a constant can be created for these too.  Much cleaner and easier to change in the future